### PR TITLE
Update index.js

### PR DESCRIPTION
**Pull request by tosih**
Loop until connection.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -11,7 +11,8 @@ const mongoHost = process.env.MONGODB_HOST || 'localhost'
 
 mongoose.connect(`mongodb://${mongoHost}/todo`)
 mongoose.connection.on('error', () => {
-  console.log('ERROR: Unable to connect to MongoDB.')
+  console.log('ERROR: Unable to connect to MongoDB... retrying')
+  mongoose.connect(`mongodb://${mongoHost}/todo`)
```
Comment (anandkumarpatel) [inline index.js]: need timeout

File: index.js [hunk 2]
```diff
@@ -11,7 +11,8 @@ const mongoHost = process.env.MONGODB_HOST || 'localhost'
 
 mongoose.connect(`mongodb://${mongoHost}/todo`)
 mongoose.connection.on('error', () => {
-  console.log('ERROR: Unable to connect to MongoDB.')
+  console.log('ERROR: Unable to connect to MongoDB... retrying')
```
Comment (anandkumarpatel) [inline index.js]: remove error, say waiting for mongo. will look better for the user

File: index.js [hunk 3]
```diff
@@ -11,7 +11,10 @@ const mongoHost = process.env.MONGODB_HOST || 'localhost'
 
 mongoose.connect(`mongodb://${mongoHost}/todo`)
 mongoose.connection.on('error', () => {
-  console.log('ERROR: Unable to connect to MongoDB.')
+  console.log('Unable to connect to MongoDB... retrying')
```
Comment (anandkumarpatel) [inline index.js]: `Unable to connect to MongoDB...` still sounds harsh? `Waiting for mongo connection`

Comment (prafulrana) [reply index.js]: lol

**Review by anandkumarpatel — APPROVED** [review]

**Pull request merged**